posthog/database_healthcheck.py

```diff
@@ -43,8 +43,10 @@ def _get_timebucket(self) -> int:
         return round(time.time() / self.time_interval)
 
     def is_postgres_connected_check(self) -> bool:
+        from posthog.models.feature_flag.flag_matching import DATABASE_FOR_FLAG_MATCHING
```

Comment:
Can this import be moved to the top level? Or do you end up with cyclic imports? If the latter, is it possible to refactor to have a non-cyclic dependency graph by moving DATABASE_FOR_FLAG_MATCHING to a separate common dependency file?

Reply:
Ah, no cyclic imports, was testing something different, forgot to move it back up, will do!

Reply:
nvm, is cyclic, will move FLAG_MATCHING elsewhere